tests/TransformerTest.php

```diff
@@ -103,4 +103,27 @@ protected function map()
             $transformer::transform(['Iwazaru' => 134, 'cl' => 212])
         );
     }
+
+    /**
+     * @dataProvider transformerOptions
+     */
+    public function testTransformerCanAccessOptions(array $options)
+    {
+        $transformer = new class([], '', $options) extends Transformer {
+            protected function map()
+            {
+            }
+        };
+
+        $this->assertSame($options, $transformer->getOptions());
+    }
+
+    public function transformerOptions(): array
+    {
+        return [
+            [[]],
+            [['foor' => 'bar']],
+            [['foor' => 'bar', 'john' => 'doe']],
+        ];
+    }
```

Comment:
We have to also add an integration test with the rest of transformers to show how it can be used, and to be sure to not have regressions about that in the future.